File: C++ Code/Basics/Mensuration/3 D/Cylinder.cpp
// WAP to calculate Volume,TSA and CSA of a Cylinder.
#include<iostream>
using namespace std;
int main(){
    float r,h;
    cout<<"Enter the base radius : ";
    cin>>r;
    cout<<"Enter the height : ";
    cin>>h;
    cout<<"Volume = "<<3.14*r*r*h<<endl;
    cout<<"Total Surface Area = "<<2*3.14*r*(r+h)<<endl;
    cout<<"Curved Surface Area = "<<2*3.14*r*h;
    return 0;
}

File: Java Code/Basics/Mensuration/Cylinder.java
import java.util.*;

public class Cylinder{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    float PI=22.0f/7.0f;
    System.out.println("For the Cylinder :-");
    System.out.print("Enter radius : ");
    float r=sc.nextFloat();
    System.out.print("Enter height : ");
    float h=sc.nextFloat();
    float LSA=2*PI*r*h;
    float TSA=2*PI*r*(r+h);
    float V=PI*r*r*h;
    System.out.println("Lateral Surface Area : "+LSA);
    System.out.println("Total Surface Area : "+TSA);
    System.out.println("Volume : "+V);
  }
}